wiki/Tournaments/o!cCO/2023/en.md

```diff
@@ -0,0 +1,560 @@
+   - In the Grand Final, the winner of the the upper bracket will only need to win a single match in order to claim the championship title. The winner of the lower bracket, however, will need to win two matches and enforce a *bracket reset* in order to clinch the championship title.
+4. Teams who can compete in the next round are determined by:
+   - In the Round of 32 and the Round of 16, each team needs to win 5 points in order to win a match. (Best of 9)
+   - In the Quarterfinals and the Semifinals, each team needs to win 6 points in order to win a match. (Best of 11)
+   - In both of the Finals weeks, each team needs to win 7 points in order to win a match. (Best of 13)
+   - Whether there are teams who are declared to win the match by default.
+   - Whether there are teams who are disqualified from the tournament.
+
+### Match regulations
+
+1. Prior to starting the match, representatives from each team must run the `!roll` command in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+   - This rule does not apply in the Qualifier lobbies.
+2. Each team has to ban **one beatmap** from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.
+   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
+3. Each team will be given a chance to pick **one warm-up beatmap** to be played in the lobby.
+   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to. 
+4. Excluding the Qualifiers, each stage of the tournament will feature a **MixedMod pool** that goes as follows:
+   - The MixedMod pool will be played with FreeMod activated.
+   - During a MixedMod pick, one player from each team will have to play the map with the Hidden mod activated, while the other player will have to play with either HardRock or Hidden+HardRock under their belt.
+5. When playing a map from the HardRock or DoubleTime pool, the Hidden modification may be optionally used by any player.
+6. In the case of a tiebreaker, the tiebreaker map will be played with the FreeMod option enabled. Players are free to play the tiebreaker map with Hidden, HardRock, or Hidden+HardRock should they wish to.
+   - Playing the tiebreaker map with a mod is *not* mandatory.
+7. The results of each match and any other relevant information regarding the match will be noted by the referee after the match has been concluded.
+
+[^qual-note-1]: Average score across all eight Qualifier beatmaps: CTB RAIMON 1,903,488, DreStarkari 1,892,753.
+[^qual-note-2]: Average score across all eight Qualifier beatmaps: Signal Graph 1,613,901, Ban Mods 1,608,756.
+[^qual-note-3]: Average score across all eight Qualifier beatmaps: Rhodes Island 1,516,625, Ad Matai Bad 1,506,365.
```

Comment:
what's so special about integer average ranks that you decided to add footnotes? I don't see any purpose for them to be in this article

Reply:
this is to indicate which seed belongs to which team mainly, as several teams are tied by their average qualifier ranks

![image](https://github.com/ppy/osu-wiki/assets/36564236/3086aed1-1d78-464a-88fa-77c3e87fd9bd)